src/Console/Command/DescribeCommand.php

```diff
@@ -155,7 +155,7 @@ private function describeRule(OutputInterface $output, string $name): void
         if ($fixer instanceof DeprecatedFixerInterface) {
             $successors = $fixer->getSuccessorsNames();
             $message = [] === $successors
-                ? 'will be removed on next major version'
+                ? 'will be removed with the next major version'
```

Comment:
I also wanted to change it at some point, but I was thinking rather about "in the" 😉. Why "with"?

Reply:
@Wirone 

Wondered myself: technically, it is already removed after the merge of this pull request, not with the commit that is tagged with the next major version. But I am not a native speaker, so I don't know for sure.

Reply:
But from the users' perspective in most cases change will take effect only after new version was released, so all the changes are provided at once, so we can treat new version similarly to huge squashed commit with all the changes since the previous version 😅.

I have feedback from native speaker (my english teacher):

> WITH sounds like the version will also be removed :wink: so I’d go with FROM or IN